File: 20151031/reverse.rb
array = []
open('data.txt') do |f|
  while l = f.gets
    array << l unless l.chomp == ''
  end
end

f = File.open('tmp.txt', 'w')
array.reverse.each { |t| f.puts t }
f.close

system('cat tmp.txt | pbcopy')
system('rm tmp.txt')

File: 20160127/reverse.py
import subprocess

with open('data.txt', encoding='utf-8') as fp_r:
    array = []
    for line in fp_r:
        if not line.rstrip() == '':
            array.append(line)

array.reverse()

with open('tmp.txt', 'w', encoding='utf-8') as fp_w:
    for line in array:
        fp_w.write(line)

subprocess.call('cat tmp.txt | pbcopy', shell=True)
subprocess.call('rm tmp.txt', shell=True)
